src/reducers/reducer-utils.ts

```diff
@@ -0,0 +1,2 @@
+export const getAuthDomain = (authDomain?: string | null): string[] =>
+  authDomain == null ? [] : [authDomain];
```

Comment:
```suggestion
export const getAuthDomain = (authDomain?: string): string[] =>
  authDomain === undefined ? [] : [authDomain];
```

Since `authdomain` is optional, can we switch the signature to this?